internal/api/v1alpha1/access_config.go

```diff
@@ -29,6 +29,13 @@ type AccessConfig struct {
 	//
 	// +kubebuilder:default:="24h"
 	MaxDuration string `json:"maxDuration"`
+
+	// AccessCommand is used to describe to the user how they can make use of their temporary access.
+	// The AccessCommand can be data templated by a Exec/PodAccessTemplate' metadata field
```

Comment:
You're not templating against the Template object though, you are passing in the Pod itself..

Reply:
Just update this comment, then merge it.. :+1:

Reply:
Not sure what the best wording to describe this. Perhaps ? 
```
The AccessCommand can reference data from a Pod ObjectMeta.
```
I do not have permissions to merge this PR as well

Reply:
Yes that wording is good…